module/cyphersystem.js

```diff
@@ -598,18 +598,18 @@ function easedRollMacro() {
 
 function hinderedRollMacro() {
   let d = new Dialog({
-    title: "Hindered Stat Roll",
-    content: `<div align="center"><label style='display: inline-block; text-align: right'><b>Hindered by: </b></label>
+    title: game.i18n.localize("CYPHERSYSTEM.HinderedStatRoll"),
+    content: `<div align="center"><label style='display: inline-block; text-align: right'><b>${game.i18n.localize("CYPHERSYSTEM.HinderedBy")}: </b></label>
```

Comment:
This needs to be capitalized, but isn’t in the string.

Reply:
Same comment as previous

Reply:
Refactored translation